content/docs/04.5-devx/05-app-profile/03-app-profile-cloning.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "App Profile Cloning"
+metaTitle: "Palette Dev Engine App Profile Cloning"
+metaDescription: "Palette Dev Engine App Profile Cloning"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Clone App Profiles
+
+Palette supports the cloning of app profiles across multiple projects. For example, clone an app profile created under the project scope to the scope of another project within the same tenant. Hence, use an app profile's clone under different project scopes for app deployment.
+
+## Use Case
+
+* Clone app profiles from system scope to project scope.
+
+
+* Clone app profiles across different project scopes under the same tenant.
+
+# Clone an App Profile
+
+To clone an App Profile:
```

Comment:
To Clone an App Profile, complete the following steps.


^ More of an active voice vs passive